content/en/users/data/storage/object-storage/_index.md

```diff
@@ -354,6 +394,30 @@ $ davix-ls --s3accesskey 'access' --s3secretkey 'secret' \
 `davix-get`, `davix-put` and `davix-del` are also available to download, store
 and delete objects from the storage.
 
+{{% alert title="Note" color="info" %}}
+The Davix Client does not support access to public buckets
+{{% /alert %}}
+
+#### Access via Python
+
+The possibility to access progammatically via S3 object storage is also quite
+important, for instance in the case of interactive computing via EGI Notebooks.
+
+When using Python for instance, [S3Fs](https://s3fs.readthedocs.io/en/latest/)
+is a practical Pythonic file interface to S3.
+
+The top-level class `S3FileSystem` holds connection information and allows typical
+file-system style operations like cp, mv, ls, du, glob, etc., as well as put/get
+of local files to/from S3.
+
+```python
+import s3fs
+fs = s3fs.S3FileSystem(anon=True,
+      client_kwargs={
+         'endpoint_url': 'https://object-store.cloud.muni.cz'
+      })
+```
```

Comment:
```suggestion
```python
from minio import Minio
from minio.error import S3Error

def main():
    client = Minio("object-store.cloud.muni.cz")

    buckets = client.list_buckets()
    for bucket in buckets:
        print(bucket.name, bucket.creation_date)

    # List all object paths in bucket
    objects = client.list_objects('test-egi-public', recursive=True)
    for obj in objects:
        print(obj.bucket_name, obj.object_name.encode('utf-8'), obj.last_modified,
              obj.etag, obj.size, obj.content_type)

    # Get a full object
    data = client.get_object('test-egi-public', 'file1.txt')
    with open('my-testfile', 'wb') as file_data:
        for d in data.stream(32*1024):
            file_data.write(d)

if __name__ == "__main__":
    try:
        main()
    except S3Error as exc:
        print("error occurred.", exc)
```
```

Reply:
arg, example for the wrong library! :D 
let me resubmit this change